dipy/viz/__init__.py

```diff
@@ -1,10 +1,13 @@
 # Init file for visualization package
 from __future__ import division, print_function, absolute_import
 
-
 from dipy.utils.optpkg import optional_package
 # Allow import, but disable doctests if we don't have fury
-fury, have_fury, _ = optional_package('fury')
+fury, has_fury, _ = optional_package(
+    'fury',
+    "You do not have FURY installed. Some visualization functions"
+    "might not work for you. For installation instructions, please visit: "
+    "https://fury.gl/")
 
 
 if have_fury:
```

Comment:
Can you update this variable `have_fury` to `has_fury`

Reply:
Updated here and elsewhere to `has_fury` to be consistent with `has_mpl` below.